bin/BuildPackages.sh

```diff
@@ -184,10 +210,36 @@ do
         *)
           run_configure_and_make
         ;;
-      esac;
+      esac
       ) || build_fail
+}
+
+      (buildpackage > >(tee "$LOGDIR/$dir.out") 2> >(tee "$LOGDIR/$dir.err" >&2) )> >(tee "$LOGDIR/$dir.log" ) 2>&1
       # end subshell
+      # remove superfluous log files if there was no error message
+      if [[ ! -s "$LOGDIR/$dir.err" ]]
+      then
+        rm -f "$LOGDIR/$dir.err"
+        rm -f "$LOGDIR/$dir.out"
+      fi
     else
       echo "$dir is not a GAP package -- no PackageInfo.g"
```

Comment:
That message actually bothers me -- I have a `.git` directory in my `pkg` dir, and it prints this "warning"... And now that there is a `log` dir, it'll also print it for that. What purpose does it server? Can't we just keep silent?